test/Groups/quotients.jl

```diff
@@ -52,14 +53,16 @@ end
    #   otherwise `PcGroup` if finite, `FPGroup` if not
    # - `maximal_abelian_quotient` with prescribed type:
    #    return a group of this type if possible
-   for T in [ PermGroup, PcGroup ]
+#  for T in [ PermGroup, PcGroup ]
+   for T in [ PermGroup, SubPcGroup ]
```

Comment:
Why not
```suggestion
   for T in [ PermGroup, PcGroup, SubPcGroup ]
```

Reply:
The point is that currently `abelian_group(PcGroup, [2, 3, 4])` is not supported.
`abelian_group` promises a result whose generators correspond to the given orders,
and the type `PcGroup` implies that the generators are a polycyclic generating sequence w.r.t. the underlying presentation.
With GAP's `IsPcGroup`, we cannot get this; and `IsPcpGroup` cannot be used because of some bugs.

(I will add a comment.)